### PR TITLE
Add optional `wrap` parameter to `BooleanInputTag::label()`

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 - Enh #220: Add `non-empty-string` psalm type of `Html::generateId()` method result (@vjik)
 - Enh #220: Add `non-empty-string|null` psalm type of `Tag::id()` method parameter (@vjik)
 - Enh #222: Bump minimal PHP version to 8.1 and refactor (@vjik)
+- New #224: Add optional `wrap` parameter to `BooleanInputTag::label()` method that allow to set label placement (@vjik)
```
Comment (samdark) [inline CHANGELOG.md:8]:
```suggestion
- New #224: Add optional `wrap` parameter to `BooleanInputTag::label()` method that allows setting label placement (@vjik)  
```

File: src/Tag/Base/BooleanInputTag.php [hunk 2]
```diff
@@ -36,12 +36,18 @@ final public function checked(bool $checked = true): static
      *
      * @param string|null $label Input label.
      * @param array $attributes Name-value set of label attributes.
+     * @param bool $wrap Whether to wrap input with label tag. If set to `false`, label will be rendered side on side
```
Comment (samdark) [inline src/Tag/Base/BooleanInputTag.php:39]: What do you mean by "side on side input"?

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,7 @@
 - Enh #220: Add `non-empty-string` psalm type of `Html::generateId()` method result (@vjik)
 - Enh #220: Add `non-empty-string|null` psalm type of `Tag::id()` method parameter (@vjik)
 - Enh #222: Bump minimal PHP version to 8.1 and refactor (@vjik)
+- New #224: Add optional `wrap` parameter to `BooleanInputTag::label()` method that allows setting label placement (@vjik)
```
Comment (arogachev) [inline CHANGELOG.md:8]:
```suggestion
- New #224: Add optional `wrap` parameter to `BooleanInputTag::label()` method that controls whether to wrap input tag with label tag or place them side by side. (@vjik)  
```

File: src/Tag/Base/BooleanInputTag.php (hunk 2 above)
Comment (arogachev) [inline src/Tag/Base/BooleanInputTag.php:39]:
```suggestion
     * @param bool $wrap Whether to wrap input with label tag. If set to `false`, label will be rendered side by side
```

File: src/Tag/Base/BooleanInputTag.php [hunk 4]
```diff
@@ -36,12 +36,18 @@ final public function checked(bool $checked = true): static
...
+     * input.
```
Comment (arogachev) [inline src/Tag/Base/BooleanInputTag.php:40]:
```suggestion
     * with input.
```

**Review by arogachev — APPROVED** [review]

File: src/Tag/Base/BooleanInputTag.php (hunk 2 above)
Comment (vjik) [reply src/Tag/Base/BooleanInputTag.php]: fixed

**Review by samdark — APPROVED** [review]

**Pull request merged**